.rubocop.yml

```diff
@@ -50,6 +50,10 @@ Layout/FirstHashElementIndentation:
 Layout/LineLength:
   Enabled: false
 
+# Adding a space after {} block improves consistency.
+Layout/SpaceAroundMethodCallOperator:
```

Comment:
Disabling this means code like this is now allowed:
```ruby
foo. bar
foo &.bar
```
https://docs.rubocop.org/rubocop/1.75/cops_layout.html#layoutspacearoundmethodcalloperator

I think this cop improves overall readability, even if it might hurt your specific case.

Reply:
@supechicken Just add a specific override of this cop in your code for your specific use?

Reply:
This cop is not configurable. Now finding a way to satisfy the check without disabling it...

Reply:
@Zopolis4 @satmandu Does the new syntax in [b661668](https://github.com/chromebrew/chromebrew/pull/11861/commits/b6616686d6107f961f41fd462b249d8da7f052be) looks better?